policy.go

```diff
@@ -575,6 +582,13 @@ func (p *Policy) AllowURLSchemesMatching(r *regexp.Regexp) *Policy {
 	return p
 }
 
+// RewriteSrc will rewrite the src attribute of a resource downloading tag
+// (e.g. <img>, <script>, <iframe>) using the provided function.
```

Comment:
This PR looks good and will be accepted.

Please expand this comment though, it's clear to me that this is a security addition and not a general rewrite addition as it's common for UGC to have any files served off a different domain name, i.e. viewing email in fastmail.com will serve all inline src files via fastmailusercontent.com as this then ensures that the browser enforcement of the domain isolation guarantees that nothing served from fastmailusercontent.com can gain access to anything on fastmail.com.

The comment today says _what_ the function does, but not _why_ one should consider using it. If you add the why, I'll merge the PR.

Reply:
The comment has been expanded on the latest commit, thank you !